modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionLocalServiceImpl.java

```diff
@@ -70,7 +72,7 @@ public ListTypeDefinition addListTypeDefinition(
 	@Override
 	public ListTypeDefinition addListTypeDefinition(
 			String externalReferenceCode, long userId,
-			Map<Locale, String> nameMap)
+			List<ListTypeEntry> listTypeEntries, Map<Locale, String> nameMap)
```

Comment:
I think we should order the parameters by following the order from service.xml. Since ListTypeDefinition does not contain the listTypeEntries, so I believe that this parameter should come at last.